eval_pipeline/utils/calculations.py

```diff
@@ -0,0 +1,174 @@
+import numpy as np
+
+def calculate_agent_bundle_values(valuation_matrix, allocation_matrix):
```

Comment:
You don't need this. 

If you have nxm V and mxn A, then C=matmul(V, A) is nxn and C_{ij} is how much agent i values the bundle assigned to agent j